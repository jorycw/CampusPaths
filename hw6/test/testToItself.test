# Simple test of breadth-first search algorithm.
# Verifies that it can find a path to itself when there are no
# edges in the graph and when there are only an edge to itself
# in the graph

CreateGraph characters

AddNode characters A

FindPath characters A A

AddEdge characters A A aEdgeA

FindPath characters A A