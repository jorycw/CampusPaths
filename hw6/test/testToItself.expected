# Simple test of breadth-first search algorithm.
# Verifies that it can find a path to itself when there are no
# edges in the graph and when there are only an edge to itself
# in the graph

created graph characters

added node A to characters

path from A to A:

added edge aEdgeA from A to A in characters

path from A to A: